src/test/scala/com/github/fulrich/testcharged/generators/ids/UuidGeneratorsUTest.scala

```diff
@@ -0,0 +1,12 @@
+package com.github.fulrich.testcharged.generators.ids
+
+import org.scalatest.{FunSuite, Matchers, OptionValues}
+
+class UuidGeneratorsUTest extends FunSuite with Matchers with OptionValues{
+  test("Single UUID generation") {
+    val generatedUUID = UuidGenerators.uuid.sample
```

Comment:
Nitpick but I would expect this val to be `generatedUuid`

Reply:
fixed